packages/plugins/rubygem-multi_xml/rubygem-multi_xml.spec

```diff
@@ -0,0 +1,86 @@
+# template: scl
+%{?scl:%scl_package rubygem-%{gem_name}}
+%{!?scl:%global pkg_name %{name}}
+
+%global gem_name multi_xml
+
+Name: %{?scl_prefix}rubygem-%{gem_name}
+Version: 0.6.0
+Release: 1%{?dist}
+Summary: A generic swappable back-end for XML parsing
+Group: Development/Languages
+License: MIT
+URL: https://github.com/sferik/multi_xml
+Source0: https://rubygems.org/gems/%{gem_name}-%{version}.gem
+
+# start specfile generated dependencies
+Requires: %{?scl_prefix_ruby}ruby(release)
+Requires: %{?scl_prefix_ruby}ruby
+Requires: %{?scl_prefix_ruby}ruby(rubygems) >= 1.3.5
+BuildRequires: %{?scl_prefix_ruby}ruby(release)
+BuildRequires: %{?scl_prefix_ruby}ruby
+BuildRequires: %{?scl_prefix_ruby}rubygems-devel >= 1.3.5
+BuildArch: noarch
+Provides: %{?scl_prefix}rubygem(%{gem_name}) = %{version}
+# end specfile generated dependencies
+
+%description
+Provides swappable XML backends utilizing LibXML, Nokogiri, Ox, or REXML.
```

Comment:
should any of these be in `Requires`?

Reply:
I checked the EPEL version and it had no requires on any of those.